.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.vscode/
```

Comment:
Not sure why codesandbox doesn't add these. You may have some git issues. If so, ping me